### PR TITLE
Clearml serving add support for triton

### PR DESCRIPTION
**Pull request by stefano-cherchi**
Add Triton Inference Server support to the clearml-serving Helm chart.

The Triton support can be disabled in the values file, if not needed.

Issue: #73

### CODE REVIEW

File: charts/clearml-serving/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: clearml-serving
 description: ClearML Serving Helm Chart
 type: application
-version: 0.1.0
+version: 0.2.0
```
Comment (valeriano-manassero) [inline charts/clearml-serving/Chart.yaml:5]: In the meanwhile I published 0.2.0. Can you pls bump up to 0.3.0?

File: charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  strategy: {}
+  template:
+    metadata:
+      annotations: {}
+      labels:
+        clearml.serving.network/clearml-serving-backend: "true"
+        clearml.serving.service: clearml-serving-triton
+    spec:
+      containers:
+        - env:
```
Comment (valeriano-manassero) [inline charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml:23]: Can you pls add
```
{{- if .Values.clearml.extraPythonPackages }}
            - name: EXTRA_PYTHON_PACKAGES
              value: '{{ join " " .Values.clearml.extraPythonPackages }}'
            {{- end }}
```
like i did for other deployments?

**Review by valeriano-manassero — CHANGES_REQUESTED** [review]
Also pls rebase :-)

File: charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml (hunk 2 above)
Comment (stefano-cherchi) [reply charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml:23]: Sure. Is this specific for the triton pod(s) or maybe it makes sense to add the same env variable to all the other deployments?

Comment (stefano-cherchi) [reply charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml:23]: Ok I see that this variable is used in three Dockerfiles: triton, serving, and statistics.  It's probably better to make the extra packages list specific for each deployment?

I'd rather use:

```yaml
{{- if .Values.clearml_serving_triton.extraPythonPackages }}
            - name: EXTRA_PYTHON_PACKAGES
              value: '{{ join " " .Values.clearml_serving_triton.extraPythonPackages }}'
            {{- end }}
```

and change `values.yaml` accordingly:

```yaml
clearml_serving_triton:
  enabled: true
  image: allegroai/clearml-serving-triton
  nodeSelector: {}
  tolerations: []
  affinity: {}
  resources: {}
  extraPythonPackages: []
```

for each of the three deployments. Wdyt?

Comment (valeriano-manassero) [reply charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml:23]: Moving `extraPythonPackages` to specific deployment is a good idea and gives more flexibility that my latest addition, so pls, go for it changing already implemented deployments.

**Review by valeriano-manassero — APPROVED** [review]

**Comment by valeriano-manassero** [comment]
\o/
ty @stefano-cherchi !

**Pull request merged**